Richmond_Library_App/templates/home.html

```diff
@@ -7,16 +7,30 @@
     <link href="{% static 'home.css' %}" rel="stylesheet" type="text/css">
   </head>
   <body>
+    <nav class="navbar navbar-expand-lg navbar-light bg-light">
+      <div class="container">
+        <a class="navbar-brand" href="#">
+          <img src="static\images\mps_logo1.png" alt=" Logo">
```

Comment:
When I run the code, the image isn't loading for me. I fixed the issue by changing the src to follow the static directory. The one that worked for me was src="{% static 'images/mps_logo1.png' %}"

Reply:
Thanks for letting me know! I had some issues as well with the image showing up. What I had was working for me, so it must have just only worked for my side.  Just made the change and assume it should work now.